hadoop-ozone/httpfsgateway/pom.xml

```diff
@@ -101,36 +101,71 @@
       <artifactId>reload4j</artifactId>
       <scope>compile</scope>
     </dependency>
-    <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
-      <scope>compile</scope>
-    </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-reload4j</artifactId>
       <scope>runtime</scope>
     </dependency>
-    <dependency>
-      <groupId>org.apache.curator</groupId>
-      <artifactId>apache-curator</artifactId>
-      <version>2.4.0</version>
-      <type>pom</type>
-    </dependency>
     <dependency>
       <groupId>org.apache.curator</groupId>
       <artifactId>curator-test</artifactId>
```

Comment:
It seems `curator-test` is unnecessary:
https://github.com/adoroszlai/hadoop-ozone/actions/runs/3996117051

(Some of) the exclusions need to be moved to `curator-framework` then.

(And don't forget to remove it from `jar-report.txt`, too.)

Reply:
thanks for checking it, I removed it from both places.